src/items/external-blocks.md

```diff
@@ -42,16 +42,16 @@ Functions within external blocks may be called by Rust code, just like
 functions defined in Rust. The Rust compiler automatically translates between
 the Rust ABI and the foreign ABI.
 
-Functions within external blocks may be variadic by specifying `...` after one
-or more named arguments in the argument list:
+A function declared in an extern block is implicitly `unsafe`. When coerced to
+a function pointer, a function declared in an extern block has type `unsafe
+extern "abi" for<'l1, ..., 'lm> fn(A1, ..., An) -> R`, where `'l1`, ... `'lm`
+are its lifetime parameters, `A1`, ..., `An` are the declared types of its
+parameters and `R` is the declared return type.
 
-```rust,ignore
-extern {
-    fn foo(x: i32, ...);
-}
-```
+It is `unsafe` to access a static item declared in an extern block, whether or
```

Comment:
"whether it's mutable or not" is what I expected to read. Not sure if that's just me or if this is incorrect grammar.

Reply:
[This](https://www.dictionary.com/browse/whether-or-not) suggests that it can go either way. I lean towards keeping it together as a single phrase "whether or not", since that is a common idiom.